addons/product/models/product_attribute.py

```diff
@@ -32,6 +32,8 @@ class ProductAttribute(models.Model):
         - Dynamically: Each variant is created only when its corresponding attributes and values are added to a sales order.
         - Never: Variants are never created for the attribute.
         Note: the variants creation mode cannot be changed once the attribute is used on at least one product.""",
+        compute="_compute_create_variant",
+        store=True,
```

Comment:
```suggestion
        store=True,
        readony=True,
```
Field should be editable as before!
Also move this attributes before `help` attribute